Makefile

```diff
@@ -113,6 +113,9 @@ else
 	$(GO) test $(GOFLAGS) -tags '$(TAGS)' -run "$(TESTS)" -bench "$(BENCHES)" -timeout $(TESTTIMEOUT) $(PKG) $(TESTFLAGS)
 endif
 
+testshort: GOFLAGS += -short
```

Comment:
I think I can add it to TESTFLAGS just as well. I copied from testrace but in that case the race flag actually influences the build so it's different.

One thing though is if I do `TESTFLAGS += -short` and I also pass some flags like`make test TESTFLAGS='-v'`, the `-short` does not show up. I don't understand why.. This is also a problem with `make testslow`: if you run make testslow TESTFLAGS='-meh' the `-v` doesn't get passed.